demisto_sdk/commands/validate/validators/RM_validators/RM106_is_contain_demisto_word.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.tools import check_text_content_contain_sub_text
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Pack
```

Comment:
I think this validation shouldn't run only on Packs, let's also check for integrations, scripts, and playbooks.